setup.cfg

```diff
@@ -114,6 +114,7 @@ filterwarnings =
     ignore:leap-second auto-update failed
     ignore:defusedxml.lxml is no longer supported:DeprecationWarning
     ignore:Using or importing the ABCs from:DeprecationWarning
+    ignore:direct construction of DocTestModulePlus has been deprecated
```

Comment:
This is actually an issue with pytest 5.4. See astropy/astropy#10039

It might be prudent to pin pytest to `<5.4` rather than ignore this warning. (That should be done on master).